.github/workflows/certora_recovery.yml

```
@@ -0,0 +1,44 @@
+name: certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rule: ['verifyRecoveryModule.sh']
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install python
+        uses: actions/setup-python@v4
+        with: { python-version: 3.11 }
```

nit: I find using Yaml objects easier to read:

```suggestion
        with:
          python-version: 3.11
```

---

Refactored 👍🏾